examples/src/App.tsx

```diff
@@ -46,18 +48,36 @@ const CustomTabPanel = ({
 
 function App() {
   const { chain } = useNetwork();
+  const { chains, switchNetwork } = useSwitchNetwork();
+
+  const [sdk, setSdk] = useState(new EndaomentSdkApi(new Configuration({ network: 'goerli' })));
```

Comment:
Rather than making this part of state, I think it would be better to simply memoize the variable.
`const sdk = useMemo(() => new EndaomentSdkApi(new Configuration({ network: chainId === 1 ? 'mainnet' : 'goerli' }), [chain?.id])`